decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -93,6 +94,12 @@ def original_proposal_url
 
         Decidim::ResourceLocatorPresenter.new(proposal.amendable).url
       end
+
+      def plain_text_body
+        proposal.body.transform_values do |v|
+          v.is_a?(Hash) ? v : convert_to_text(v)
```

Comment:
Should we also apply the same approach for the machine translations?

If I add machine translations for the proposal and export, they still have HTML in them, here's a screenshot of the exported excel file:

![Proposals export machine translations](https://user-images.githubusercontent.com/864340/209147149-54886796-2ef6-4a8c-981f-9e284bafacb5.png)

Reply:
Yes I agree there is no reason to keep HTML tags on machine_translations and not in body

I convert the PR to draft state and work on it asap

Reply:
Is it okay for you if I reuse the same body content for `en`, `machine_translation[:es]` and `machine_translation[:ca]` in `proposal_serializer_spec.rb` ? 
The idea is to keep the spec file light, but I can also create new content for each field to be sure it works on each field

Reply:
Sure, do as you see the best.

I think in the code itself you could so a recursion with a similar approach you are already doing, e.g.

```ruby
def convert_to_plain_text(value)
  return value.transform_values { |v| convert_to_plain_text(v)  } if value.is_a?(Hash)
    
  convert_to_text(value)
end
```

And then pass e.g. `proposal.body` to that method.